features/image-optimization/svg-to-raster-format-conversion.md

```diff
@@ -0,0 +1,15 @@
+# SVG to raster format conversion
+
+ImageKit.io allows you to convert SVG images to raster formats by using the [format parameter (f)](../image-transformations/resize-crop-and-other-transformations.md#format---f).
+
+To perform this conversion, you need to specify the desired raster format as `f-<raster_format>`. For example: `f-png` or `f-jpg` etc.
+
+{% hint style="info" %}
+If you enable automatic format conversion or use `f-auto`, the SVG images will not be converted to raster formats and will be delivered as SVG files only.
```

Comment:
```suggestion
If you enable automatic format conversion or use `f-auto` in the image URL, SVG images will not be converted to raster formats and will be delivered as SVG files only. Specifying `f-<raster_format>` in the URL is the only way to convert SVGs to raster formats.
```

Reply:
Also, add a link to the `Automatic format conversion` page

Reply:
https://github.com/imagekitio/docs-keep-a-changelog/pull/237#discussion_r1437996069